components/SplitsDisplay.vue

```diff
@@ -66,6 +66,7 @@
           <split-display :split="split"
                          v-for="(split, key) in splits"
                          :key="`split-${key}`"
+                         :splitIndex=key
```

Comment:
HTML attributes without quotes are technically valid but very ugly
```suggestion
                         :splitIndex="key"
```